src/components/Composer/Post/Update/index.tsx

```diff
@@ -293,18 +331,31 @@ const NewUpdate: FC = () => {
           <ReferenceSettings />
           {publicationContent && <Preview />}
         </div>
-        <div className="ml-auto pt-2 sm:pt-0">
-          <Button
-            disabled={isLoading}
-            icon={isLoading ? <Spinner size="xs" /> : <PencilAltIcon className="w-4 h-4" />}
-            onClick={createPost}
-          >
-            Post
-          </Button>
-        </div>
+        {!isAudioPost && (
+          <div className="ml-auto pt-2 sm:pt-0">
+            <Button
+              disabled={isLoading}
+              icon={isLoading ? <Spinner size="xs" /> : <PencilAltIcon className="w-4 h-4" />}
+              onClick={createPost}
+            >
+              Post
+            </Button>
+          </div>
+        )}
       </div>
       <div className="px-5">
         <Attachments attachments={attachments} setAttachments={setAttachments} isNew />
+        {isAudioPost && (
+          <div className="flex justify-end pt-4">
+            <Button
+              disabled={isLoading}
+              icon={isLoading ? <Spinner size="xs" /> : <PencilAltIcon className="w-4 h-4" />}
+              onClick={createPost}
+            >
+              Post
+            </Button>
+          </div>
+        )}
```

Comment:
**question:** why duplicate post button?

Reply:
the post button is moved down as per audio post design. Have created common comp for Button and reused.

Reply:
I think its better to use same button placement for images, audio and videos